gulpfile.js

```diff
@@ -396,10 +398,94 @@ gulp.task('zip:edge', zipTask('edge'))
 gulp.task('zip:opera', zipTask('opera'))
 gulp.task('zip', gulp.parallel('zip:chrome', 'zip:firefox', 'zip:edge', 'zip:opera'))
 
+
+/*
+ * Brave
+ */
+
+/*
+ * ToDo(ryanml) - Write a method to convert simple paths to a Regex obj
+ * then a simple mapping can be created for these replacements.
+ * Ex (psuedo-ish):
+ * const replacements = [
+ *   {
+ *     target: 'actions',
+ *     replace: 'ui/app/store/actions'
+ *   }
+ * ]
+ *
+ * replacements.map((item) => {
+ *   .pipe(
+ *     replace(convertToRegex(item.target), fmtPath(item.replace))
+ *   )
+ * })
+ */
+
+const overrideDirs = [
+  'ui/app/**/*',
+  'app/scripts/**/*'
+]
+
+const braveRoot = '~/brave/'
```

Comment:
nit: recommend renaming this to `const bravePrefix` for greppability

Reply:
done